client.c

```diff
@@ -1362,6 +1365,26 @@ ssize_t read(int fd, void *buf, size_t count) {
 		}
 
 		return count;
+	} else {
+		int t = get_timer_from_fd(fd);
+		if (t >= 0) {
+			if (count >= sizeof(timers[t].expired)) {
+				do {
+					if (timers[t].expired) {
+						memcpy(buf, &timers[t].expired, sizeof(timers[t].expired));
+						timers[t].expired = 0;
```

Comment:
Just a nit, after `sizeof` there should be space to follow the coding style.

Reply:
Ok. I'm sorry. I use sizeof like a normal function (regarding the Coding style).